telegram/warnings.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+# pylint: disable=cyclic-import
```

Comment:
do we need this here if it's already defined globally?

Reply:
yea it didn't work ill remove